From abd83550d389cc396d25aafb1807538933b778f3 Mon Sep 17 00:00:00 2001
From: mldn <lixiang15025187785@163.com>
Date: Fri, 22 Nov 2019 10:50:49 +0800
Subject: [PATCH] Change Admin.java

---
 Admin.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Admin.java b/Admin.java
index b9501e4..ef27c2a 100644
--- a/Admin.java
+++ b/Admin.java
@@ -1,4 +1,5 @@
 import java.util.List;
 public class Admin(){
 	private List<Emp> emps;
+	private String adminId;
 }
\ No newline at end of file
-- 
2.22.0.windows.1

